waffle/apps.py

```diff
@@ -4,6 +4,7 @@
 class WaffleConfig(AppConfig):
     name = 'waffle'
     verbose_name = 'django-waffle'
+    default_auto_field = 'django.db.models.AutoField'
```

Comment:
if I install `django-waffle` (once released with this change) and install it on Django 3.2, would it be in my interest to use `BigAutoField` even for this package?

Sorry if I'm getting something obvious very wrong here.

Reply:
You shouldn't need to change this - `AutoField` is still the default for projects that have been upgraded to 3.2, and the difference between the two is pretty minimal aside from `BigAutoField` giving a larger range of values when you get to very high ID numbers.

I left it as `AutoField` to keep things simple and avoid generating migrations, but I'd be happy to change to `BigAutoField` if people would prefer.

Reply:
That makes sense. I should have clarified I'm asking from the point of view of a fresh Django 3.2 project, rather than an upgrade.

Reply:
You shouldn't need to change it either way